packages/features/insights/server/trpc-router.ts

```diff
@@ -1239,6 +1239,7 @@ export const insightsRouter = router({
               select: UserSelect,
             },
           },
+          distinct: ["userId"],
```

Comment:
This is due team org members gets double membership to parent and to child team.